universal-application-tool-0.0.1/test/FunctionalTest.java

```diff
@@ -86,6 +94,68 @@ public void createPerson() {
     assertThat(contentAsString(result)).contains("person John with ID:", "created.");
   }
 
+  @Test
+  public void createProgram() {
+    final DbRepository repo = app.injector().instanceOf(DbRepository.class);
+    Program program = new Program();
+    program.name = "program one";
+    program.version = 1L;
+    program.object = ImmutableMap.of("key", "value");
+    repo.insertProgram(program).toCompletableFuture().join();
+    Program p = repo.lookupProgram("program one", 1L).toCompletableFuture().join().get();
+    assertThat(p.version).isEqualTo(1L);
+    assertThat(p.object).containsEntry("key", "value");
+  }
+
+  @Test
+  public void createApplicant() {
+    final DbRepository repo = app.injector().instanceOf(DbRepository.class);
+    Applicant applicant = new Applicant();
+    applicant.id = 1L;
+    applicant.object =
+        ImmutableMap.of("nestedObject", ImmutableMap.of("foo", "bar"), "secondKey", "value");
+    repo.insertApplicant(applicant).toCompletableFuture().join();
+    Applicant a = repo.lookupApplicant(1L).toCompletableFuture().join().get();
+    assertThat(a.id).isEqualTo(1L);
+    assertThat(a.object).containsAllEntriesOf(applicant.object);
+  }
```

Comment:
Nit: please use whitespace to separate statements in test methods into arrange, act, assert sections as is feasible/reasonable for a given test

Reply:
Done.